private/buf/bufformat/testdata/proto2/message/v1/message_options.golden.proto

```diff
@@ -37,7 +37,7 @@ message Foo {
     foo: 1
     bar: 2
     baz: 3
-    buzz: <
+    buzz: {
```

Comment:
Replacing `<` with `{` feels like a nice cleanup. Are there any inputs where `<` would be preferable?